libpod/oci_missing.go

```diff
@@ -194,6 +195,11 @@ func (r *MissingRuntime) SupportsKVM() bool {
 	return false
 }
 
+// Features returns nil since this is a missing runtime
+func (r *MissingRuntime) Features() *features.Features {
+	return nil
```

Comment:
This should still return an error, so I recommend (*features.Features, error) - the missing runtime can have methods called on it and we want to avoid null pointer panics